src/views/GameView.vue

```diff
@@ -442,6 +443,12 @@ export default {
       'resolvingSeven',
       'hasGlassesEight',
     ]),
+    isSpectating() {
+      return this.$router.currentRoute.value.name === ROUTE_NAME_SPECTATE;
+    },
```

Comment:
TODO: show which one-off's and counters are on the stack so that spectators can see which things have been played while waiting for an opponent to counter

Reply:
https://github.com/cuttle-cards/cuttle/issues/353